tests/release_tests/test_launch/utils.py

```diff
@@ -92,3 +98,56 @@ def update_dict(original, updated):
             update_dict(original[key], value)
         else:
             original[key] = value
+
+
+def create_config_files():
+    """Create a launch-config.yml and launch-agent.yml."""
+    launch_config = yaml.load_all(
+        open("wandb/sdk/launch/deploys/kubernetes/launch-config.yaml")
+    )
+    launch_config_patch = yaml.load_all(
+        open("tests/release_tests/test_launch/launch-config-patch.yaml")
+    )
+    final_launch_config = []
+    for original, updated in zip(launch_config, launch_config_patch):
+        document = dict(original)
+        update_dict(document, dict(updated))
+        final_launch_config.append(document)
+    yaml.dump_all(
+        final_launch_config,
+        open("tests/release_tests/test_launch/launch-config.yml", "w+"),
+    )
+    launch_agent = yaml.load(
+        open("wandb/sdk/launch/deploys/kubernetes/launch-agent.yaml")
+    )
+    launch_agent_patch = yaml.load(
+        open("tests/release_tests/test_launch/launch-agent-patch.yaml")
+    )
+    launch_agent_dict = dict(launch_agent)
+    update_dict(launch_agent_dict, dict(launch_agent_patch))
+    yaml.dump(
+        launch_agent_dict,
+        open("tests/release_tests/test_launch/launch-agent.yml", "w+"),
+    )
```

Comment:
Long-term we'll want to think about whether this will become bloated, with new tests requiring more and more generated config files.

We probably want more specific functions, `generate_local_k8s_launch_config`, `generate_local_k8s_launch_agent`, etc, but I don't think we need to worry about that until we have more tests and need more modular control of file generation. Ideally we'd want something that only makes config files that are required for the specific tests being run, in the event someone is only running one or two.

Oh also, we want to use helm long term, so this patching method will hopefully be obsolete soon anyways.